apiExamples/basicGroup.html

```diff
@@ -1,6 +1,6 @@
 <auro-radio-group>
-  <span slot="legend">Form label goes here</span>
-  <auro-radio id="radio28" label="Yes" name="radioDemo" value="yes"></auro-radio>
-  <auro-radio id="radio29" label="No" name="radioDemo" value="no"></auro-radio>
-  <auro-radio id="radio30" label="Maybe" name="radioDemo" value="maybe"></auro-radio>
+  <span slot="legend">Are you willing to volunteer resources?</span>
+  <auro-radio id="radio4" label="Yes" name="radioDemo" value="yes">Volunteering is AWESOME!</auro-radio>
+  <auro-radio id="radio5" label="No" name="radioDemo" value="no">Sorry, I do not have the time or resources</auro-radio>
+  <auro-radio id="radio6" label="Maybe" name="radioDemo" value="maybe">Please send me more information</auro-radio>
```

Comment:
Why did all the examples content change? It doesn't seem connected to the change for `optionSelected`.